example/src/pages/LinePage.tsx

```diff
@@ -132,6 +134,24 @@ const Line: React.FC<{}> = () => {
     });
   }, [state]);
 
+  const selectLine = useCallback((lineId: number) => {
+    const deselectLineIndex = state.findIndex((item) => item.showPointHandle);
+    dispatch({
+      type: 'UPDATE_SET_OPTIONS',
+      index: deselectLineIndex,
+      data: {
+        showPointHandle: false,
+      }
+    });
+    dispatch({
+      type: 'UPDATE_SET_OPTIONS',
+      index: lineId,
+      data: {
+        showPointHandle: true,
+      }
+    });
```

Comment:
Just one small update request... in the example, when a line is considered to not be selected, let's hide the handle too as the usability is a little confusing there (as you can drag the line but not reselect it with that part, so this proposal is easiest). 

```suggestion
    dispatch({
      type: 'UPDATE_SET_OPTIONS',
      index: deselectLineIndex,
      data: {
        showPointHandle: false,
        showMoveHandle: false
      }
    });
    dispatch({
      type: 'UPDATE_SET_OPTIONS',
      index: lineId,
      data: {
        showPointHandle: true,
        showMoveHandle: true
      }
    });
```